e2e_tests/tests/config.py

```diff
@@ -17,6 +17,14 @@
 TF1_GPU_IMAGE = "determinedai/environments:cuda-10.0-pytorch-1.4-tf-1.15-gpu-067db2b"
 TF2_GPU_IMAGE = "determinedai/environments:cuda-10.1-pytorch-1.4-tf-2.2-gpu-067db2b"
 
+if os.environ.get('TF1_CPU_IMAGE') is not None:
```

Comment:
Here's one other closely related thing I threw in: the ability to override the defaults in this file too, with environment variables.